composer.json

```diff
@@ -17,6 +17,12 @@
 	"require-dev": {
 		"phpunit/PHPUnit": "^5.7"
 	},
+	"autoload": {
+        "psr-4": {
+            "SilverStripe\\IFrame\\": "code/",
+            "SilverStripe\\IFrame\\Tests\\": "tests/"
```

Comment:
not for modules; Modules are rarely installed in root, and autoload-dev is root only. This means if you add iframe to your project, and ran test suite, test classes won't be psr-4 autoloaded.